prefs.js

```diff
@@ -868,6 +857,6 @@ function init() {
 function buildPrefsWidget() {
     let settings = new Settings();
     let widget = settings.widget;
-    widget.show_all();
+    if (widget.show_all) widget.show_all();
```

Comment:
As per having `<requires lib="gtk" version="4.0"/>` I assume this is not needed, we can just avoid calling it now.

Reply:
Ah yes, can you tell this was the first line I changed to get this going :wink: